### PR TITLE
[IMP] mail: introduce Thread/currentPartnerLongTypingTimer

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2831082

### CODE REVIEW

File: addons/mail/static/src/models/thread.js [hunk 1]
```diff
@@ -1174,6 +1160,17 @@ registerModel({
...
             return clear();
         },
+        /**
+         * @private
+         * @returns {Timer}
+         */
+        _computeCurrentPartnerLongTypingTimer() {
+            return new Timer(
+                this.messaging,
+                () => this.async(() => this._onCurrentPartnerLongTypingTimeout()),
```
Comment (seb-odoo) [inline addons/mail/static/src/models/thread.js:1170]: remove this.async

**Review by seb-odoo — COMMENTED** [review]
ok except remove async

File: addons/mail/static/src/models/thread.js (hunk 1 above)
Comment (alexkuhn) [reply addons/mail/static/src/models/thread.js]: Done

File: addons/mail/static/src/models/thread.js [hunk 2]
```diff
@@ -57,6 +57,7 @@ registerModel({
              */
             this._currentPartnerLastNotifiedIsTyping = undefined;
             /**
+<<<<<<< HEAD
```
Comment (seb-odoo) [inline addons/mail/static/src/models/thread.js:60]: ?

**Review by seb-odoo — APPROVED** [review]

**Comment by alexkuhn** [comment]
Moved to grouped PR: https://github.com/odoo/odoo/pull/90569

**Pull request closed without merging**